bindings/python/.evergreen/test.sh

```diff
@@ -29,14 +29,23 @@ if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
 elif [ "Darwin" = "$(uname -s)" ]; then
     export PYMONGOCRYPT_LIB=${MONGOCRYPT_DIR}/nocrypto/lib/libmongocrypt.dylib
     PYMONGOCRYPT_LIB_CRYPTO=${MONGOCRYPT_DIR}/lib/libmongocrypt.dylib
-    MACOS_VER=$(sw_vers -productVersion)
-    PYTHONS=(
-          "/Library/Frameworks/Python.framework/Versions/3.8/bin/python3"
-          "/Library/Frameworks/Python.framework/Versions/3.9/bin/python3"
-          "/Library/Frameworks/Python.framework/Versions/3.10/bin/python3"
-          "/Library/Frameworks/Python.framework/Versions/3.11/bin/python3"
-          "/Library/Frameworks/Python.framework/Versions/3.12/bin/python3"
-          )
+    MACOS_ARCH=$(uname -p)
+    if [[ $MACOS_ARCH =~ "arm" ]]; then
```

Comment:
We don't test on non-arm anymore so I think we can remove this if/else.

Reply:
Updated